sdk/keyvault/keyvault-certificates/tsconfig.eslint.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Thanks! My brain didn't clicked. I like the "extends" idea.

Reply:
I pushed an update with the extends idea.

Reply:
Did you push though? 😂

Reply:
I'm kinda having trouble deciding between having this extra file everywhere vs. just adding the samples to the compilation (e.g. the main d.ts). We could exclude the built samples from the package with our `files` array in package.json. Thoughts?

Reply:
I like that last idea!

Reply:
Ok not so fast, since we have this new samples methodology, our samples use this:

    import { CertificateClient } from "@azure/keyvault-certificates";

Which means that adding samples to our tsconfig.json causes a circular dependency.

Reply:
> Did you push though?

Yep! Though from the main page of this PR it doesn't look like. But you can see it here: https://github.com/Azure/azure-sdk-for-js/pull/7136/files#diff-817b367d5432401b0fa286a104239283R2